packages/core/src/app/checkout/CheckoutStepHeader.tsx

```diff
@@ -29,10 +29,10 @@ const CheckoutStepHeader: FunctionComponent<CheckoutStepHeaderProps> = ({
     type,
 }) => {
     return (
-        <a
+        <div
             className={ classNames(
                 'stepHeader',
-                { 'is-readonly': !isEditable }
+                { 'is-clickable': isEditable && !isActive }
```

Comment:
Are we going to change the class name?

Reply:
@animesh1987 I think we need to.

Previously, a step header is `a`, which has a pointer cursor by default. `is-readonly` specifies a default cursor for customer step, which cannot be edited, `sign out` only.

Now, a step header is a div, so `is-clickable` provides a pointer cursor for editable shipping and billing steps, and leaves customer step as non-editable like before.